Chandra/cmd_states/cmd_states.py

```diff
@@ -393,14 +399,16 @@ def get_state0(date=None, db=None, date_margin=10, datepar='datestop'):
         db = Ska.DBI.DBI(dbi='sybase', server='sybase', user='aca_read',
                          database='aca')
 
-    # Date for which cmd_states are certainly reliable
-    definitive_date = DateTime(time.time() - date_margin * 86400.,
-                               format='unix').date
-    if date is None or date > definitive_date:
-        date = definitive_date
-    else:
+    if date is not None:
```

Comment:
Thought does that really mean that a `date_margin` of None is equivalent to a `date_margin` of 0?  Would logic to do that have been less confusing?  Eh.  It was good to remember what this code does anyway.